guides/doc-Release_Notes/topics/foreman-3.8.0.adoc

```diff
@@ -4,142 +4,163 @@ A full list of changes is available on https://projects.theforeman.org/issues?se
 
 == Foreman
 
-* Add "last_checkin" attribute to Entitlements Template - https://projects.theforeman.org/issues/30690[#30690]
+* pass:[Some events are not visible even being triggered] - https://projects.theforeman.org/issues/36796[#36796]
+* pass:[CVE-2022-4130: Blind SSRF via Referer header] - https://projects.theforeman.org/issues/36768[#36768]
+* pass:[Add "last_checkin" attribute to Entitlements Template] - https://projects.theforeman.org/issues/30690[#30690]
 
 === API
 
-* GraphQL: Incorrect totalCount when querying with first parameter - https://projects.theforeman.org/issues/36509[#36509]
-* Unable to access API using non-admin users. - https://projects.theforeman.org/issues/36449[#36449]
+* pass:[GraphQL: Incorrect totalCount when querying with first parameter] - https://projects.theforeman.org/issues/36509[#36509]
```

Comment:
What is `pass:[...]` for?

Reply:
https://github.com/theforeman/foreman-documentation/pull/2427